modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -266,16 +270,37 @@ sda.enkf.multisite <- function(settings, obs.mean, obs.cov, Q = NULL, restart=F,
         )
       })
     
-    if(t==1)  inputs <- out.configs %>% map(~.x[['samples']][['met']]) # for any time after t==1 the met is the splitted met
+    if (t == 1)  inputs <- out.configs %>% map(~.x[['samples']][['met']]) # for any time after t==1 the met is the splitted met
     #-------------------------------------------- RUN
    
     PEcAn.remote::start.model.runs(settings, settings$database$bety$write)
+    
+    if (forceRun == TRUE)
```

Comment:
I want to push back on this part again. I really don't think it's a good idea to bring a "quick fix" into the mainline.  I don't know what might be the scenario for pecan to leave some jobs not submitted , but if there is it needs to be fix inside `PEcAn.remote::start.model.runs` function.

Reply:
@para2x you should chat with @bailsofhay this week about why we have this "patch" to see how we should proceed

Reply:
@mdietze  and I just had a conversation about this. He said to keep the bandaid for now, but to go ahead and keep it for this merge, just so we can get my code for multisite added to the SDA code. This will be for the next PR